nixos/community.tt

```diff
@@ -23,7 +23,7 @@
 please open a <a
 href="https://github.com/NixOS/nixpkgs/pulls"><strong>pull request on
 GitHub</strong></a>, or <strong>send a patch</strong> to the <a
-href="[%root%]nixos/support.html#mailing-list"><tt>nix-devel</tt> mailing list</a>. If you
+href="[%root%]nixos/support.html#mailing-list"><tt>Discourse</tt> mailing list</a>. If you
```

Comment:
Do we actually want people posting patches on Discourse? TBH I would prefer to remove the whole "send a patch" thing.

Reply:
As long as there are people willing to apply the patches it doesn't hurt

Reply:
The rule still applies that a github maintainer needs to be assigned to each packages and modules